frame/staking/src/tests.rs

```diff
@@ -5233,18 +5246,19 @@ fn proportional_ledger_slash_works() {
 	ledger.active = unit;
 	ledger.total = unit * 4 + value;
 	// When
-	assert_eq!(ledger.slash(slash, 0, 0), slash - 43);
+	assert_eq!(ledger.slash(slash, 0, 0), slash - 5);
```

Comment:
why is this changed from 43 to 5?

Reply:
With rounding up each chunk will be slashed a bit more than before thus improving the accuration, the total slash will never exceed the expected amount though due to:
https://github.com/paritytech/substrate/blob/4c83ee0a406939f1393d19f87675e1fbc49e328d/frame/staking/src/lib.rs#L630